contracts/external/actions/AccountAction.sol

```diff
@@ -209,8 +212,7 @@ contract AccountAction is ActionGuards {
             FreeCollateralExternal.checkFreeCollateralAndRevert(redeemer);
         }
 
-        emit nTokenSupplyChange(redeemer, currencyId, balance.netNTokenSupplyChange);
```

Comment:
Why did you remove this event?

Reply:
It's emitted already inside BalanceHandler, so this is a duplicate emit, was causing some issues with our Dune Dashboards.